controllers/index.js

```diff
@@ -0,0 +1,36 @@
+const {
```

Comment:
Eventually you'll want to split these controllers into separate files, i.e. articles.controllers.js, users.controllers.js etc. Not something to do in this pull request but I'd address it as soon as you're able to.